transports/spvwallet/client_factory_adapter.go

```diff
@@ -44,17 +35,9 @@ func (bf *walletClientFactory) CreateAdminClient() (users.AdminWalletClient, err
 
 // CreateWithXpriv returns UserWalletClient as spv-wallet-go-client instance with given xpriv.
 func (bf *walletClientFactory) CreateWithXpriv(xpriv string) (users.UserWalletClient, error) {
-	serverUrl, signRequest := getServerData()
-
-	userWalletClient, err := walletclient.New(
-		walletclient.WithXPriv(xpriv),
-		walletclient.WithHTTP(serverUrl),
-		walletclient.WithSignRequest(signRequest),
-	)
+	serverUrl, _ := getServerData()
```

Comment:
IMO, since we removed the explicit `signRequest` param, we could remove it from `getServerData` function, and generally from `-web-backend` config.